retrieval/relabel_test.go

```diff
@@ -173,6 +173,27 @@ func TestRelabel(t *testing.T) {
 				"d": "58",
 			},
 		},
+		{
+			input: clientmodel.LabelSet{
+				"a":  "foo",
+				"b1": "bar",
+				"b2": "baz",
+			},
+			relabel: []*config.RelabelConfig{
+				{
+					Regex:       &config.Regexp{*regexp.MustCompile("^(b.*)")},
+					Replacement: "bar_${1}",
+					Action:      config.RelabelMapLabels,
+				},
+			},
+			output: clientmodel.LabelSet{
+				"a":      "foo",
+				"b1":     "bar",
+				"b2":     "baz",
+				"bar_b1": "bar",
+				"bar_b2": "baz",
+			},
+		},
```

Comment:
The intend is to do mass-conversion from `__meta_<labelname>` to `<labelname>`, right? So a test doing exactly that would be good.